main.tex

```diff
@@ -16,8 +16,36 @@
 }
 
 \begin{document}
+\section{Facebooks Absicht/Gründe für Internet.org}
+Facebook, ein soziales Netzwerk, wurde 2004 auf den Markt gebracht und seit da an ist es, und das dahinterstehende Unternehmen, facebook inc, stetig gewachsen. 
+Doch da die jährliche Anzahl neuer Kabelanschlüsse, für das Internet, global gesehen seit 2008 immer weniger geworden ist,
```

Comment:
In der westlichen Welt :)

Reply:
Vlt. `...seit 2008 gesunken sind`
und `Anmeldung neuer Kabelanschlüsse`?

Reply:
Zum 1. Kommentar: schlägst du vor "global" mit "westliche Welt" auszutauschen?

Reply:
> Zum 1. Kommentar: schlägst du vor "global" mit "westliche Welt" auszutauschen?

Irgendetwas in der Art, es soll halt daraus hervorgehen das in der westlichen Welt Internet schon sehr weit verbreitet ist und deshalb nicht viel Wachstumspotential besteht.